shell/server/core.js

```diff
@@ -147,6 +148,46 @@ class NotificationHandle {
   }
 }
 
+class PersistentSealedUiViewImpl {
+  constructor(token, membraneRequirements) {
+    this._token = token;
+    // TODO: implement SystemPersistent.addRequirements() in a way that collects requirements, and
+    // then saves them appropriately in save().
+    this._membraneRequirements = membraneRequirements || [];
+  }
+
+  save(params) {
+    // Create a new child apiToken for the user whose session attempted to save this capability.
+    // The token should specify this.token as the parent token, and needs to gain an additional membrane
+    // requirement (TODO):
+    // * if params.sealFor is a user, then the user must still have access to the grain that offered
+    //   this cap.  Today, this is handled by offer() in SessionContextImpl.
+    const res = inMeteor(() => {
+      const owner = _.extend({}, params.sealFor);
+      const grainId = this._token.grainId;
+
+      if (owner.user) {
+        // Copy the owner's save label for this capability, if available, to be the new title.
+        if (this._token && this._token.owner && this._token.owner.grain &&
+            this._token.owner.grain.saveLabel && this._token.owner.grain.saveLabel.defaultText) {
+          owner.user.title = this._token.owner.grain.saveLabel.defaultText;
+        }
+
+        // Denormalize this grain's app metadata into owner.user.
+        owner.user.denormalizedGrainMetadata = globalDb.getDenormalizedGrainInfo(grainId);
+      }
+
+      const ret = makeChildTokenInternal(this._token._id, owner, this._membraneRequirements, grainId);
```

Comment:
Hmmm.  This is where things get kinda weird.

The saved child token kind-of-isn't-really-a-SealedUiView.  It's a full UiView, which should have a membrane requirement that it can't be restored unless the SealedUiView that led to its creation is still valid.

A SealedUiView isn't actually a UiView; it's an opaque handle associated with a UiView that the Sandstorm shell will give you or redeem into a proper UiView if you offer it to a client with the (invisible and implicit) permission "is a human user".

Given that: is using `parentToken` still the right thing to do here, if in practice the relation doesn't match what `parentToken` is documented as meaning?  Is there an alternative expression that would be more consistent?